resources/scripts/dev/commands/alignment_command.lua

```diff
@@ -6,10 +6,15 @@ local function cycleAlignment()
     local alignment = Game.party.getAlignment()
     if alignment == Game.PartyAlignment.Good then
         alignment = Game.PartyAlignment.Neutral
+        Game.party.setQBit(Game.QBits.LightPath, false)
```

Comment:
This assumes that qbits and alignment are in sync, which is not guaranteed. Let's assign both qbits in all if branches.

Reply:
Well they must be synched if current game state is not broken.
But you are right here, must set all consistently just in case.